modules/dbnd/src/dbnd/_core/plugin/dbnd_plugins_mng.py

```diff
@@ -25,3 +29,4 @@ def register_dbnd_plugins():
 def register_dbnd_user_plugins(user_plugin_modules):
     for plugin_module in user_plugin_modules:
         pm.register(_load_module(plugin_module, "plugin:%s" % plugin_module))
+        logger.info("Plugin %s loaded..." % plugin_module)
```

Comment:
Can we try retrieving version through plugin's `__version__` attribute? Worst case it doesn't exist

Reply:
sure, we can

Reply:
fixd